transform/mattermost-analytics/models/intermediate/product/active_users/int_user_active_days_legacy_telemetry.sql

```diff
@@ -0,0 +1,45 @@
+{{
+    config({
+        "materialized": "incremental",
+        "incremental_strategy": "merge",
+        "merge_update_columns": ['received_at_date'],
+        "unique_key": ['daily_user_id'],
+        "cluster_by": ['received_at_date'],
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+
+with tmp as (
+    select
+        cast(received_at as date) as received_at_date,
+        cast(timestamp as date) as activity_date,
+        server_id,
+        user_id
+    from
+        {{ ref('stg_mattermost2__tracks') }}
+    where
+        -- Exclude items without user ids, such as server side telemetry etc
+        user_id is not null
```

Comment:
Should we mention somewhere more prominently (like the folder name) that this work only includes webapp events (did a quick check for 06/16/2023 and out of 1.5M events about a third are eliminated by this check - prob same for non-legacy source as well)